ballerina/Module.md

```diff
@@ -0,0 +1,185 @@
+## Overview
+The [Ballerina](https://ballerina.io/) connector for [CData Connect](https://cloud.cdata.com/docs/JDBC.html) allows you to programmatically access all of the CData Connect applications, databases, APIs, and services across an organization via the Java Database Connectivity (JDBC) API using [Ballerina](https://ballerina.io/). 
+For detailed information on working with each data source, see [Data Sources](https://cloud.cdata.com/docs/Data-Sources.html).
+CData Connect supports a wide range of standard DDL commands, SQL commands, and SQL functions to query data sources. 
+For reference information on all the CData Connect SQL commands (i.e., DDL, DML, and query syntax), see the [SQL Reference](https://cloud.cdata.com/docs/SQL-Reference.html).
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+### To connect to CData Connect Cloud
+
+* Create a [CData Connect Cloud](https://cloud.cdata.com) account.
+* Obtain the CData Connect Cloud `username` and `password` from the CData Connect Cloud dashboard. 
+    * `<username>` is the email address of the user. For example `user@cdata.com`.
+    * `<password>` is a personal access token (PAT) that you generate from the **User Profile** page. For instructions on how to create a PAT to authenticate, see [Personal Access Tokens](https://cloud.cdata.com/docs/User-Profile.html#personal-access-tokens). The CData Connect Connector uses Basic AuthScheme.
+* Make sure to go to the [Connections](https://cloud.cdata.com/docs/Connections.html) tab 
+in the CData Connect Cloud dashboard and set up any connection you need to work with the data sources. 
+You can use the **Connections** tab to configure a data source that contains the data you want to work with. 
+For more information on working with each data source, see [Data Sources](https://cloud.cdata.com/docs/Data-Sources.html).
+Use the **Connection Name** and **Data Source Name** to write your SQL queries.
+
+### To connect to CData Connect Server
+
+* Run the CData Connect Server.
+* Obtain the CData Connect Server `username` and `password`.
+    * `<username>` is the username you use to login to CData Cloud Connect Server.
+    * `<password>` is the password you use to login to CData Cloud Connect Server.
+* Obtain the `<hostname>` and `<port>` where your CData Connect Server is up and running.
+* You need to specify the `<url>` as `jdbc:cdata:connect:URL=http://<hostname>:<port>/rest.rsc;`.
```

Comment:
Should this be mentioned here? The URL is something users should construct when configuring the connector.

Reply:
Ah yes. I agree. This line is removed from here since it's already mentioned when configuring.